src/Idris/REPL.hs

```diff
@@ -1474,6 +1474,17 @@ process fn (PPrint fmt width t)
             ty' = normaliseC ctxt [] ty
         iPrintResult =<< renderExternal fmt width (pprintDelab ist tm)
 
+
+process fn Quit = iPrintError "Unimplemented pattern for command Quit"
```

Comment:
I would use something like `iPrintError "Command ':quit'" is current unsupported` instead.